lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -5,6 +5,15 @@
 module EVSS
   module DisabilityCompensationForm
     class Form526ToLighthouseTransform
+      TOXIC_EXPOSURE_CAUSE_MAP = {
+        NEW: 'My condition was caused by an injury or exposure during my military service.',
+        WORSENED: 'My condition existed before I served in the military, but it got worse because of my military ' \
+                  'service.',
+        VA: 'My condition was caused by an injury or event that happened when I was receiving VA care.',
+        SECONDARY: 'My condition was caused by another service-connected disability I already have. (For example, I ' \
+                   'have a limp that caused lower-back problems.)'
+      }.freeze
```

Comment:
Not a big deal, but i wonder if the VSR's need the example text.

Reply:
Good point. Got the go-ahead from Shannon to shorten it. 👍